packages/medusa-payment-stripe/src/api/index.ts

```diff
@@ -1,10 +1,22 @@
+import { authenticate } from "@medusajs/medusa"
 import { Router } from "express"
+import { getStripePayments } from "../controllers/get-payments"
 import hooks from "./hooks"
 
 export default (container) => {
   const app = Router()
 
   hooks(app)
 
+  app.options(`/admin/stripe-payments/:order_id`, authenticate())
```

Comment:
Suggestion: /admin/orders/:order_id/stripe-payments just to be explicit that it is a medusa order id as opposed to a payment order id